src/Manipulators/Brightness.php

```diff
@@ -34,6 +34,10 @@ public function run(Image $image)
      */
     public function getBrightness()
     {
+        if (null === $this->bri) {
```

Comment:
A separate `if` block isn't necessary. You can just add `null === $this->bri || ` to the `if` block below.